app/src/routes/login/components/sso-links.vue

```diff
@@ -103,12 +103,24 @@ const errorFormatted = computed(() => {
 		align-items: center;
 		justify-content: center;
 		width: var(--theme--form--field--input--height);
+		height: var(--theme--form--field--input--height);
```

Comment:
Perfect opportunity for an `aspect-ratio: 1` 😉

Reply:
Gotcha!